common/src/test/scala/uk/ac/wellcome/message/MessagePointerTest.scala

```diff
@@ -0,0 +1,37 @@
+package uk.ac.wellcome.message
+
+import org.scalatest._
+import uk.ac.wellcome.utils.JsonUtil._
+import java.net.URI
+
+import uk.ac.wellcome.exceptions.GracefulFailureException
+import uk.ac.wellcome.s3.S3ObjectLocation
+
+import scala.util.{Failure, Success}
+
+class MessagePointerTest extends FunSpec with Matchers {
```

Comment:
These are all testing creating a `MessagePointer` from a `URI` which I think is not necessary anymore